### PR TITLE
Added devise

### PR DESCRIPTION
**Pull request by BrendaPenn**
Added and installed devise for the login.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+  # If true, extends the user's remember period when remembered via cookie.
+  # config.extend_remember_period = false
+
+  # Options to be passed to the created cookie. For instance, you can set
+  # secure: true in order to force SSL only cookies.
+  # config.rememberable_options = {}
+
+  # ==> Configuration for :validatable
+  # Range for password length.
+  config.password_length = 6..128
```
Comment (phlipper) [inline config/initializers/devise.rb]: We should follow the [NIST Digital Identity Guidelines](https://pages.nist.gov/800-63-3/sp800-63-3.html) here and make the minimum length at least `8`. 

Some layperson explanations are available [here](https://auth0.com/blog/dont-pass-on-the-new-nist-password-guidelines/).

There are also a number of other recommendations in there we should create issues for to attach to future work :)

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+  # config.reset_password_keys = [:email]
+
+  # Time interval you can reset your password with a reset password key.
+  # Don't put a too small interval or your users won't have the time to
+  # change their passwords.
+  config.reset_password_within = 6.hours
+
+  # When set to false, does not sign a user in automatically after their password is
+  # reset. Defaults to true, so a user is signed in automatically after a reset.
+  # config.sign_in_after_reset_password = true
```
Comment (phlipper) [inline config/initializers/devise.rb]: A good security practice would be to set this to `false`.

File: app/models/member.rb [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 # ActiveRecord Member user
 class Member < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
```
Comment (phlipper) [inline app/models/member.rb]: I recommend enabling `confirmable` unless there is a strong case against it.

**Comment by phlipper** [comment]
It looks like there are some test updates that need to be addressed in this branch. This will also need a rebase on top of the latest master.

File: config/initializers/devise.rb (hunk 2 above)
Comment (BrendaPenn) [reply config/initializers/devise.rb]: Thank you Phil! I have been busy and am working on making these changes and modifying the tests.

**Comment by colinloretz** [comment]
Excited to see where we're at with this one tonight at the meeting. We might be able to get it ready to go!

**Comment by BrendaPenn** [comment]
Pull request #98 is the up to date one with correct test modifications for Devise.

**Pull request closed without merging**